contrib/macdeploy/fancy.plist

```diff
@@ -22,7 +22,7 @@
 			<integer>370</integer>
 			<integer>156</integer>
 		</array>
-		<key>Pocketcoin-Qt.app</key>
+		<key>Bitcoin-Qt.app</key>
```

Comment:
I think need to restore Pocketcoin in such places

Reply:
I agree, it turned out ugly) in any case, this code has not yet been run to successfully build the version for macos